api/appsettings.go

```diff
@@ -0,0 +1,18 @@
+package api
+
+// AppSettings is the structure of an app's settings.
+type AppSettings struct {
+	// Owner is the app owner. It cannot be updated with AppSettings.Set(). See app.Transfer().
+	Owner string `json:"owner,omitempty"`
+	// App is the app name. It cannot be updated at all right now.
+	App string `json:"app,omitempty"`
+	// Created is the time that the application settings was created and cannot be updated.
+	Created string `json:"created,omitempty"`
+	// Updated is the last time the application settings was changed and cannot be updated.
+	Updated string `json:"updated,omitempty"`
+	// UUID is a unique string reflecting the application settings in its current state.
+	// It changes every time the application settings is changed and cannot be updated.
+	UUID string `json:"uuid,omitempty"`
+	// Maintenance determines if the application is taken down for maintenance or not.
+	Maintenance bool `json:"maintenance"`
```

Comment:
This falls into the same problem I'm seeing with #73. Maintenance mode must be a *bool or else if someone does a `deis config:set` while maintenance mode is on, it will disable maintenance mode because a bool's "zero" value is false.